pandas/_libs/missing.pyx

```diff
@@ -248,6 +248,31 @@ cdef bint checknull_with_nat_and_na(object obj):
     return checknull_with_nat(obj) or obj is C_NA
 
 
+@cython.wraparound(False)
+@cython.boundscheck(False)
+def is_float_nan(values: ndarray) -> ndarray:
+    """
+    True for elements which correspond to a float nan
```

Comment:
So this is to check `float('nan')` and not necessarily `np.nan`?

Reply:
`isinstance(np.nan, float)` is True. If `np.nan` isn't a float - at least a lot of things in pandas would become strange...

Reply:
Gotcha thanks just confirming